COMMENTS.md

```diff
@@ -0,0 +1,160 @@
+#### address_book.test.js
+
+Address Book\
```

Comment:
Need to remove the `\`

Reply:
I used it to make a line break, is there other way?

Reply:
Probably one \ to many in the escape sequence?